demo/tests/tests_integration.py

```diff
@@ -219,6 +219,11 @@ def test_get(self):
             self.assertIn(
                 access['label'], [obj.label for obj in get_accesses()]
             )
+            self.assertIn('preview_as', access)
+            if access['id'] == 'robot':
+                self.assertEqual(access['preview_as'], 'TABLE')
```

Comment:
Maybe use `PreviewMode.TABLE` here instead of the actual value. Not sure though.